source/math/FloatTransform3D.ooc

```diff
@@ -208,7 +208,7 @@ FloatTransform3D: cover {
 	}
 	transformAndProject: func ~FloatPoint2D (point: FloatPoint2D, focalLength: Float) -> FloatPoint2D {
 		transformedWorldPoint := this * FloatPoint3D new(point x, point y, focalLength)
-		this project(transformedWorldPoint, focalLength)
+		focalLength == 0.0f ? FloatPoint2D new(transformedWorldPoint x, transformedWorldPoint y) : this project(transformedWorldPoint, focalLength)
```

Comment:
> We don't have "isNaN" either.

What do you mean? You can check if a number is `NaN` by comparing it to itself. If `value` is a `Float` and `value != value`, then `value` is `NaN`. Are you suggesting we should have a static function for this?